src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.video.unit.spec.jsx

```diff
@@ -1071,6 +1071,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
         },
       };
       mockFacilitiesFetch('vha_442', [facility]);
+      mockFacilityFetch('vha_442', facility);
```

Comment:
This was interesting, due to the changes I made to avoid some 404 errors, sometimes the facility used here was the default "Fake" facility info, because we can't fully wait for that call to finish on the list page (video appointments don't display facility info on the cards). This makes sure to set the facility to the right data no matter if it uses the call from the list page or an individual one on the details page.